wavepool/models.py

```diff
@@ -32,4 +32,14 @@ def teaser(self):
 
     @property
     def source_divesite_name(self):
-        return 'Industry Dive'
+        """ Return the real divesite source name for the newspost's source
+        """
+        source_dive = 'Industry Dive'
+        for dive_source in DIVESITE_SOURCE_NAMES:
+            url_parts = self.source.split('/')
+            dive_domain = url_parts[2]
+            domain_parts = dive_domain.split('.')
+            dive_domain = domain_parts[1]
```

Comment:
All of this string parsing is potentially fragile - take a look and see if there is a more robust way to do this